edm.yaml

```diff
@@ -1,44 +1,49 @@
 ---
+options:
+  getSyntax: False
+  exposePODMembers: True
+
 components:
 # A component is a POD designed to be embedded in another POD,
 # see the Particle or Jet PODs.
 # Components cannot be stored in a collection
 
   fcc::LorentzVector:
-    Mass : float
-    Px : float
-    Py : float
-    Pz : float
+    mass : float
+    px : float
+    py : float
+    pz : float
 
   fcc::Point:
-    X : float
-    Y : float
-    Z : float
+    x : float
+    y : float
+    z : float
 
+  # really need to expose some function f(Cellid) -> position
   fcc::BareHit:
-    Cellid : unsigned
-    Energy : float
-    Time : float
-    Bits : unsigned
+    cellId : unsigned long long
+    energy : float
+    time : float
+    bits : unsigned
 
   fcc::BareCluster:
-    Energy : float
-    Time : float
+    energy : float
+    time : float
     position : fcc::Point
-    Bits : unsigned
+    bits : unsigned
 
   fcc::BareParticle:
-    Type: int
-    Charge: int
-    Status: unsigned
-    Vertex: fcc::Point
-    P4: fcc::LorentzVector
-    Bits: unsigned
+    type: int
```

Comment:
In the meeting Benedikt mentioned that type is needed in PAPAS and that it is not exactly a PdgId.. Should we add another member for the Id or live with the potential ambiguity?